### PR TITLE
Fix(Codeowners): Remove/replace pg (patch)

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
### Removed
- PG as codeowner

### Changed
- author to be Clinical Genomics

## Review
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Inform to VJ, MR, HO, HS

### CODE REVIEW

**Review by Vince-janv — APPROVED** [review]
Do not merge before 14/7 🥺

**Comment by patrikgrenfeldt** [comment]
@Vince-janv @Mropat @henningonsbring @henrikstranneheim I removed myself and added VJ as codeowner for the CG repo

**Pull request merged**
